temporal-serviceclient/src/main/java/io/temporal/serviceclient/SimpleSslContextBuilder.java

```diff
@@ -47,21 +47,23 @@
           // gRPC requires http2 protocol.
           ApplicationProtocolNames.HTTP_2);
 
-  private final InputStream keyCertChainInputStream;
-  private final InputStream keyInputStream;
+  private InputStream keyCertChain;
+  private InputStream key;
   private TrustManager trustManager;
   private boolean useInsecureTrustManager;
   private String keyPassword;
 
   /**
-   * @param keyCertChainInputStream - an input stream for an X.509 client certificate chain in PEM
-   *     format.
-   * @param keyInputStream - an input stream for a PKCS#8 client private key in PEM format.
+   * @param keyCertChain - an input stream for an X.509 client certificate chain in PEM format.
+   * @param key - an input stream for a PKCS#8 client private key in PEM format.
```

Comment:
May be rename to "privateKey" or "privateKeyPem?

Reply:
I kept parameter names same as in the netty SslOntextBuilder, which can also take files. I don't think we need to support files as input streams are more generic and it's easy to create FileInputStream from file path. Given that we don't support multiple inputs we can simplify the naming.

Reply:
I would prefer to keep names consistent with the SslContextBuilder to avoid confusion.